dashboard/db/migrate/20170519134008_drop_prize_columns_from_user.rb

```diff
@@ -0,0 +1,17 @@
+class DropPrizeColumnsFromUser < ActiveRecord::Migration[5.0]
+  def change
+    remove_index :users, [:prize_id, :deleted_at]
+    remove_index :users, [:teacher_prize_id, :deleted_at]
+    remove_index :users, [:teacher_bonus_prize_id, :deleted_at]
+
+    remove_column :users, :prize_id, :integer
+    remove_column :users, :teacher_prize_id, :integer
+    remove_column :users, :teacher_bonus_prize_id, :integer
```

Comment:
Yes. But since it really doesn't matter (even if we were to down the migration (incredibly unlikely), the columns would continue to be unused), I'm not going to go through the trouble of adding them.